packages/tests/web-test-runner.config.js

```diff
@@ -4,218 +4,45 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {createRequire} from 'module';
+import {wtrConfig} from './wtr-config.js';
 import {playwrightLauncher} from '@web/test-runner-playwright';
-import {fromRollup} from '@web/dev-server-rollup';
-import {createSauceLabsLauncher} from '@web/test-runner-saucelabs';
-import {legacyPlugin} from '@web/dev-server-legacy';
-import {resolveRemap} from './rollup-resolve-remap.js';
-import {prodResolveRemapConfig, devResolveRemapConfig} from './wtr-config.js';
 
-const mode = process.env.MODE || 'dev';
-if (!['dev', 'prod'].includes(mode)) {
-  throw new Error(`MODE must be "dev" or "prod", was "${mode}"`);
-}
+const LOCAL = 'local';
 
-let resolveRemapConfig;
-if (mode === 'prod') {
-  console.log('Using production builds');
-  resolveRemapConfig = prodResolveRemapConfig;
-} else {
-  console.log('Using development builds');
-  resolveRemapConfig = devResolveRemapConfig;
-}
+const requestedBrowsers = process.env.BROWSERS?.trim() || LOCAL;
 
+/***
+ * Keep browsers on separate lines to make it easier to comment out
+ * individual browsers.
+ */
 const browserPresets = {
-  // Default set of Playwright browsers to test when running locally.
-  local: [
-    'chromium', // keep browsers on separate lines
-    'firefox', // to make it easier to comment out
-    'webkit', // individual browsers
-  ],
-
-  // Browsers to test during automated continuous integration.
-  //
-  // https://saucelabs.com/platform/supported-browsers-devices
-  // https://wiki.saucelabs.com/display/DOCS/Platform+Configurator
-  //
-  // Many browser configurations don't yet work with @web/test-runner-saucelabs.
-  // See https://github.com/modernweb-dev/web/issues/472.
-  sauce: [
-    'sauce:Windows 10/Firefox@78', // Current ESR. See: https://wiki.mozilla.org/Release_Management/Calendar
-    'sauce:Windows 10/Chrome@latest-3',
-    'sauce:macOS 10.15/Safari@latest',
-    // 'sauce:Windows 10/MicrosoftEdge@18', // needs globalThis polyfill
-  ],
-  'sauce-ie11': ['sauce:Windows 7/Internet Explorer@11'],
+  chromium: {args: ['--js-flags=--expose-gc', '--enable-precise-memory-info']}, //
+  firefox: {},
+  webkit: {},
 };
 
-let sauceLauncher;
-
-function makeSauceLauncherOnce() {
-  if (!sauceLauncher) {
-    const user = (process.env.SAUCE_USERNAME || '').trim();
-    const key = (process.env.SAUCE_ACCESS_KEY || '').trim();
-    if (!user || !key) {
-      throw new Error(
-        'To test on Sauce, set the SAUCE_USERNAME' +
-          ' and SAUCE_ACCESS_KEY environment variables.'
-      );
-    }
-    sauceLauncher = createSauceLabsLauncher({
-      user,
-      key,
-    });
-  }
-  return sauceLauncher;
+const browsers = [];
+if (browserPresets[requestedBrowsers] !== undefined) {
+  browsers.push(
+    playwrightLauncher({
+      product: requestedBrowsers,
```

Comment:
Note, the previous interface for `BROWSERS` was a comma-separated list of browsers (hence the plural) parsed by 
```js
const browsers = (process.env.BROWSERS || 'preset:local')
  .split(',')
  .map(parseBrowser)
  .flat();
```

I don't think this code works with a list of browsers anymore, just one browser or `local`. I don't know everyone's personal workflows for running tests, but I'd normally err on the side of not changing stuff like that. @justinfagnani @sorvell opinions? Do you use this env variable?

Reply:
I'll add the multi browser support back in :)

My main goals here with the different configs is two fold:
- split saucelabs configurations from local configurations
- avoid recursion in a config file

I kinda took a liberty with the `preset:` keyword. They're all presets so that distinction didn't land for me. Plus when you use it `BROWSERS=preset:local`, we just substring `preset:` out anyways.
```
const preset = browser.substring('preset:'.length);
const entries = browserPresets[preset];
```

If we wanna keep the `preset:` prefix, i can add that support back in too!

Reply:
I've added back this functionality
`preset:local` and `preset:sauce` work again
user can define lists of browsers  ie `BROWERS=chromium,firefox`